### DISCUSSION TITLE
如何根据TLE推导卫星轨道

### DISCUSSION THREAD

**Discussion by xploveht** [IO-Aerospace-software-engineering/Astrodynamics#135] [General]
Api

**Comment by sylvain-guillet** [comment]
Hello,

You have two solutions :

1. You can call ToStatevector method with given tdb epoch from your TLE object. Example : https://github.com/IO-Aerospace-software-engineering/Astrodynamics/blob/main/IO.Astrodynamics.Tests/TLETests.cpp#L230

2. You can use the TLEIntegrator into your propagator like this : https://github.com/IO-Aerospace-software-engineering/Astrodynamics/blob/main/IO.Astrodynamics.Tests/PropagatorTests.cpp#L339

I hope it can help you.
